Tests/FoundationInternationalizationTests/Formatting/DateFormatStyleTests.swift

```diff
@@ -15,7 +15,7 @@ import TestSupport
 #endif
 
 #if canImport(FoundationEssentials)
-@testable import FoundationEssentials
```

Comment:
We made `_FoundationInternals`'s internal functions available for both `FoundationEssentials` and `FoundationInternationalization` by adding its source files as symbolic links at both target's path (#117). A side effect of this approach is that we can no longer import both targets with `@testable import` -- the test sees the declaration of these internal functions from both targets, so callsites would get an "ambiguous use of \<function\>" error when it attempts to use them.  

So we have to stop importing other targets with `@testable import`. This is probably desired anyways

Reply:
Dammit... what if we want to test `internal` method on `FoundationEssentials`?

Reply:
We move to-be-tested to `_FoundationInternal` so that test-importing either module grants access to those internal functions? I wouldn't worry too much about it now as long as the code and tests are structured well